tests/unit/coordinator_shard_map.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <sstream>
+#include <string>
+
+#include "common/types.hpp"
+#include "coordinator/shard_map.hpp"
+#include "gtest/gtest.h"
+#include "storage/v3/id_types.hpp"
+#include "storage/v3/property_value.hpp"
+#include "storage/v3/schemas.hpp"
+
+namespace memgraph::coordinator::tests {
+TEST(ShardMap, Parse) {
+  std::string input = R"(4
+property_1
+property_2
+property_3
+property_4
+3
+edge_type_1
+edge_type_2
+edge_type_3
+2
+label_1
+1
+primary_property_name_1
+string
+4
+[asdasd]
+[qweqwe]
+[bnm]
+[tryuryturtyur]
+label_2
+3
+property_1
+string
+property_2
+int
+primary_property_name_2
+InT
+2
+[first,1 ,2]
+[     second   ,-1,  -9223372036854775808]
+)";
```

Comment:
From what I understand we don't have a way to put secondary labels here?

Reply:
Actually that is a bug, good catch!